Image Gallery and Nav/assets/css/nav.scss

```diff
@@ -0,0 +1,113 @@
+nav {
+    margin-top: 0;
+    padding: 20px 20px;
+    display: flex;
+    justify-content: space-between;
+    position: sticky;
+    top: 0;
+    background-color: rgba($color: #000000, $alpha:0.5);
+    color: ghostwhite;
+
+    z-index: 999;
+
+    .nav-brand {
+        font-weight: 700;
+        text-transform: uppercase;
+        letter-spacing: 5px;
+        font-size: 20px;
+    }
+
+    .nav-item,
+    .nav-brand {
+        cursor: pointer;
+    }
+
+    .nav-items {
+        width: 60%;
+        margin-left: auto;
+        display: flex;
+        justify-content: space-evenly;
+        align-items: center;
+        font-weight: lighter;
+        font-size: 14px;
+
+        .nav-item {
+            transition: 0.4s all ease;
+
+            &:hover {
+                transform: scale(1.4) rotateY(-360deg);
+            }
+        }
+    }
+
+    .toggle {
+        display: none;
+        pointer-events: none;
+
+        div {
+            background-color: ghostwhite;
+            height: 3px;
+            width: 30px;
+            transition: all 0.3s ease;
+        }
+
+        div:not(:first-of-type) {
+            margin-top: 6px;
+        }
+    }
+
+    .toggle.active {
+        div:nth-of-type(1) {
+            transform: rotate(45deg) translateY(13px);
+        }
+
+        div:nth-of-type(2) {
+            opacity: 0;
+        }
+
+        div:nth-of-type(3) {
+            transform: rotate(-45deg) translateY(-13px);
+        }
+    }
+}
+
+@media only screen and (max-width:780px) {
+    nav {
+        overflow: hidden;
+
+        .toggle {
+            display: block !important;
+            margin-left: auto;
+
```

Comment:
@king-11 
You may consider adding a border with (color as white and appropriate opacity) to it so that it looks more like a raised button and becomes noticeable, otherwise this is also fine and everything else looks perfect to me.

Reply:
ok thanks made the changes :+1: